Sources/MapboxCoreNavigation/FeedbackEvent.swift

```diff
@@ -14,12 +14,12 @@ public class FeedbackEvent: Codable {
         coreEvent = CoreFeedbackEvent(timestamp: Date(), eventDictionary: dictionary ?? [:])
     }
     
-    func update(type: FeedbackType, source: FeedbackSource, description: String?) {
+    func update(type: String, subtype: String?, source: FeedbackSource = .user, description: String?) {
```

Comment:
This method is now working with both `FeedbackType` and `PassiveNavigationFeedbackType`, so I guess it's easier just to pass raw values for type and subtype.

Reply:
We may have a common protocol with `description` and `subtypeDescription` properties (or maybe have better naming). But that's optional